src/alpine/3.12/WithNode/amd64/Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.12-helix
```

Comment:
This needs to be a non-Helix image, just as the 3.9-WithNode Dockerfile is doing.

```suggestion
FROM mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.12
```

So you'd need to make a copy of https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/main/src/alpine/3.9/amd64/Dockerfile and modify it for 3.12.